bundle.js

```diff
@@ -0,0 +1,501 @@
+	card10.text = "Pay Hospital Fees of $100";
+	card10.action = function(player) {
+		player.money -= 100;
+	};
+
+	var card11 = Object.create(Card);
+	card11.text = "Pay School Fees of $50";
+	card11.action = function(player) {
+		player.money -= 50;
+	};
+
+	var card12 = Object.create(Card);
+	card12.text = "Receive $25 Consultancy Fee";
+	card12.action = function(player) {
+		player.money += 25;
+	};
+
+	var card13 = Object.create(Card);
+	card13.text = "You are assessed for street repairs - $40 per house, $115 per hotel";
+	card13.action = function(player) {
+		var housesCount = 0;
+		var hotelsCount = 0;
+		for (var prop in player.properties) {
+			if (player.properties.hasOwnProperty(prop)) {
+			// or if (Object.prototype.hasOwnProperty.call(obj,prop)) for safety...
+				// TODO - link up correctly w/ property object
+				housesCount += player.properties[prop].numberOfHouses;
+				if (player.properties[prop].hasHotel) {
+					hotelsCount += 1;
+				}
+			}
+		}
+		var total = 40 * housesCount + 115 * hotelsCount;
+		player.money -= total;
+	};
+
+	var card14 = Object.create(Card);
+	card14.text = "You have won second prize in a beauty contest - collect $10";
+	card14.action = function(player) {
+		player.money += 10;
+	};
+
+	var card15 = Object.create(Card);
+	card15.text = "You inherit $100";
+	card15.action = function(player) {
+		player.money += 100;
+	};
+
+	var card16 = Object.create(Card);
+	card16.text = "From sale of stock you get $50";
+	card16.action = function(player) {
+		player.money += 50;
+	};
+
+	var card17 = Object.create(Card);
+	card17.text = "Holiday Fund matures - Receive $100";
+	card17.action = function(player) {
+		player.money += 100;
+	};
+
+	var communityChestDeck = [card1, card2, card3, card4, card5, card6, card7, card8, card9, card10, card11, card12, card13, card14, card15, card16, card17];
+
+
+	var chance1 = Object.create(Card);
+	chance1.text = "Advance to Go (Collect $200)";
+	chance1.action = function(player) {
+		player.position = 0;
+	};
+
+	var chance2 = Object.create(Card);
+	chance2.text = "Advance to Illinois Ave.";
+	chance2.action = function(player) {
+		player.position = 24;
+	};
+
+	var chance3 = Object.create(Card);
+	chance3.text = "Advance token to the nearest Utility. If unowned, you may buy it from the Bank. If owned, throw dice and pay owner a total of ten times the amount thrown.";
+	chance3.action = function(player) {
+		if (player.position > 11) {
+			// move to water works
+			player.position = 28;
+		} else {
+			// move to electric company
+			player.position = 12;
+		}
+
+		// TODO - pay owner 10x dice
+	};
+
+	var chance4 = Object.create(Card);
+	chance4.text = "Advance token to the nearest Railroad and pay owner twice the rental to which he/she is otherwise entitled. If Railroad is unowned, you may buy it from the Bank.";
+	chance4.action = function(player) {
+		if (player.position >= 35) {
+			// Reading RR
+			player.position = 5;
+			player.money += 200;
```

Comment:
I don't think directly changing player.position is the best way to handle these chance movements - is there a way we can do a player.move(position) or something?